v1.html

```diff
@@ -1,7 +1,27 @@
 <html>
 <head>
+<!--
+Hello! If you're reading this, perhaps you're wondering what this
+file is doing and why your Riot is using it.
+
+In short, this allows Riot to isolate potentially unsafe encrypted
+attachments into their own origin, away from your Riot. See
+https://github.com/matrix-org/usercontent/blob/master/README.md
+for more info, or https://github.com/vector-im/riot-web/blob/master/README.md#configjson
+if you'd like to host your own.
```

Comment:
let's add a link to bug 6173 so people can track progress on replacing it with iframes